### ISSUE TITLE
Android generator

### ISSUE THREAD

**Issue by fcostantini** [rodrigopolo/minEmoji#18]
Hi, is it possible for you to provide a new version to generate a png sheet with Android emojis? The one in sprite_gen makes a 7 column sheet, and the order of the emojis is different. I see you have a valid sheet in minEmoji/img/ but that one is missing a lot of emojis (they are replaced with their Apple counterpart I think). I tried to adapt your last version (3) but I failed miserably.  

Thanks, and I understand if you are too busy to do it.

**Comment by rodrigopolo** [comment]
Hi, I really want to, but I can't find any source for the Android emoji images, if you know any source I'll be more than glad to add them, BTW I have added the iOS 10 / macOS 10.12 new emojis:

https://github.com/rodrigopolo/minEmoji/tree/master/minEmoji_iOS10

**Comment by fcostantini** [comment]
I managed to do it. I got the emojis from https://github.com/googlei18n/noto-emoji (an old version because I don't like the current emojis) and the missing flags from my own phone fonts. Then I renamed the images and used your gen3. The result wasn't perfect cause I couldn't resize the source pngs correctly (they were 128x120) but it worked out for my purposes.  

Here is the sheet: https://github.com/jgalat0/emojipasta-gen/blob/master/img/android.png. If you want I can upload the folder with all the renamed emojis.
